pkg/cmd/condition/describe.go

```diff
@@ -0,0 +1,183 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package condition
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+	"text/template"
+
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/cli/pkg/formatted"
+	"github.com/tektoncd/cli/pkg/printer"
+	"github.com/tektoncd/cli/pkg/validate"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+const templ = `{{decorate "bold" "Name"}}:	{{ .Condition.Name }}
+{{decorate "bold" "Namespace"}}:	{{ .Condition.Namespace }}
+{{- if ne .Condition.Spec.Description "" }}
+{{decorate "bold" "Description"}}:	{{ .Condition.Spec.Description }}
```

Comment:
Should there perhaps be some kind of character limit for the description?

Reply:
In describing I thought we should show complete, in the list we are showing limited

Reply:
I think there should always be a limit, but, if it's a truly an issue, we can address it later.